### DISCUSSION TITLE
Minimum Python version and C++ standard

### DISCUSSION THREAD

**Discussion by tbennun** [spcl/dace#1770] [Ideas]
After a discussion in the DaCe meeting, we propose to change the minimum requirements of DaCe, starting from 1.0.x onwards. They will be:
* Python 3.9
* C++17

Are there any objections?

**Comment by edopao** [comment]
No objections from GT4Py side. C++17 is already a requirement for Gridtools and the support for Python 3.8 and 3.9 will be soon dropped in GT4Py.

**Comment by romanc** [comment]
No objection from the NDSL side: Python 3.11 is already a requirement for us.
